setup.py

```diff
@@ -25,7 +25,7 @@
         "gidgethub",
         "appdirs",
         # GitHub client
-        "github3.py",
+        "github3.py>=1.3.0",
```

Comment:
Why greater on equal here and pin in requirements.txt?

Reply:
Because I want to express the minimum required version (the one that has the check suite api).
requirements.txt pins the known good versions.